### PR TITLE
readme modified and diagram added

### PR DESCRIPTION
**Pull request by andreazzabr**

### CODE REVIEW

**Review by hsy3418 — COMMENTED** [review]
I am thinking for the final infrastructure diagram, might need to add more details: IGW,NGW, route tables and security group details.

**Comment by hsy3418** [comment]
I had done a bit of poc on storage and also discussed with Caio, instead of S3, using EFS will be a better solutions for Wordpress storage, it is easy to be mounted in ECS, do you want to change your diagram?

**Review by caiocezart — CHANGES_REQUESTED** [review]
EFS is also within the vpc on the subnets

Also, could you please add some documentation?

**Comment by andreazzabr** [comment]
new diagram added
readme.md updated

File: README.md [hunk 1]
```diff
@@ -1 +1,12 @@
-2020-feb-project1-group3
\ No newline at end of file
+2020-feb-project1-group3
+
+# Diagram 
+
+[PR1-grp3](images/PR1-grp3.png)
```
Comment (caiocezart) [inline README.md]: attach the image to the documentation so you don't need to click to see it

`<some-alt-text>`

Comment (caiocezart) [reply README.md]: and please give me a few words for each one of the elements of the diagram

**Review by caiocezart — CHANGES_REQUESTED** [review]

**Comment by caiocezart** [comment]
this fix #3

**Review by caiocezart — APPROVED** [review]

**Pull request merged**